backend/src/graphql/types/type/Location.gql

```diff
@@ -17,7 +17,7 @@ type Location {
   distanceToMe: Int
 }
 
-# This is not smart - we need one location for everything - use the same type everywhere!
+"This is not smart - we need one location for everything - use the same type everywhere!"
```

Comment:
Same here? This comment is added to the documentation.
![image](https://github.com/user-attachments/assets/979e3246-41fb-44bb-98af-f9157b863cfe)